src/docs/product/dashboards/custom-dashboards/index.mdx

```diff
@@ -12,30 +12,32 @@ This feature is available only if your organization is on either a Business or T
 
 </Note>
 
-Custom Dashboards are shared across your organization, and users can view each other’s custom dashboards by selecting them from the dropdown menu in the top right. Note that any changes you make in the [default dashboard](/product/dashboards/) (called Dashboard) are global, and will appear for users across the entire organization.
+Custom Dashboards are shared across your organization, and users can view each other’s custom dashboards in the management view. Note that any changes you make are global, and will appear for users across the entire organization.
 
-![Dashboard navigation.](dashboard_cropped.png)
+## Manage Dashboards
+When you have either a Business or Trial Plan, you'll land on a view to manage dashboards instead of the [default dashboard](/product/dashboards/). From here, you can:
+- Create a dashboard
+- Duplicate a dashboard
+- Delete a dashboard
 
-Clicking either "Create Dashboard" or "Edit Dashboard" will take you into edit mode, which allows you  to edit at both the dashboard and widget level. When you edit an existing dashboard, you can tell that you’re in edit mode because the widgets are lightly grayed out.
+![Manage mode.](dashboard-management.png)
 
-![Edit mode.](dashboard_edit.png)
-
-At the dashboard level, you can:
+Clicking "Create Dashboard" will immediately take you to a new dashboard where you can immediately begin adding widgets. When you're creating a new dashboard or edit an existing dashboard, the widgets will be slightly grayed out. From here, you can:
```

Comment:
I've added a break between the sentence about creating a dashboard and the sentence about creating or editing because the list that comes after it pertains to both create and edit. It's confusing to have it preceded by a paragraph that starts with just creating.

```suggestion
Clicking "Create Dashboard" takes you to a new dashboard where you can immediately begin adding widgets. 

When you're creating a new dashboard or editing an existing one, the widgets are slightly grayed out. From here, you can:
```